build.psm1

```diff
@@ -1104,11 +1108,11 @@ function Start-PSPackage {
     Write-Verbose "Packaging RID: '$Runtime'; Packaging Configuration: '$Configuration'" -Verbose
 
     # Make sure the most recent build satisfies the package requirement
-    if (-not $Script:Options -or                                ## Start-PSBuild hasn't been executed yet
-        -not $Script:Options.CrossGen -or                       ## Last build didn't specify -CrossGen
-        $Script:Options.Runtime -ne $Runtime -or                ## Last build wasn't for the required RID
-        $Script:Options.Configuration -ne $Configuration -or    ## Last build was with configuration other than 'Release'
-        $Script:Options.Framework -ne "netcoreapp1.0") ## Last build wasn't for CoreCLR
+    if (-not $Script:Options -or                                                    ## Start-PSBuild hasn't been executed yet
+        ((-not $Script:Options.CrossGen) -AND -not ($Runtime -match "-x86")) -or    ## Last build didn't specify -CrossGen OR CrossGen check skipped since it is not supported for x86
```

Comment:
I'm assuming `Start-PSBuild -CrossGen` for win7-x86 runtime will throw error?
I recommend change `Start-PSBuild` to skip calling `Start-CrossGen` when rutnime matches -x86, and don't change the if block here. So that we always require `-CrossGen` specified when creating a package, but that parameter does nothing for x86.